package.json

```diff
@@ -115,17 +114,18 @@
     "karma-webpack": "5.0.0",
     "lint-staged": "7.3.0",
     "minami": "^1.1.1",
-    "mocha": "^9.1.4",
+    "mocha": "^8.4.0",
+    "nyc": "^15.1.0",
     "prettier": "^1.17.1",
     "sinon": "^1.14.1",
     "sinon-chai": "^2.7.0",
     "terser-webpack-plugin": "^1.3.0",
-    "ts-loader": "^5.0.0",
+    "ts-loader": "^9.4.2",
     "tslint": "^5.16.0",
     "typescript": "^3.4.5",
-    "webpack": "^4.33.0",
-    "webpack-cli": "^3.3.3",
-    "webpack-stream": "^5.2.1"
+    "webpack": "^5.75.0",
```

Comment:
buncha polyfills got dropped in webpack5 (see https://github.com/stellar/js-stellar-sdk/issues/732#issuecomment-1065897225), so make sure it all still works :sweat_smile: